common/djangoapps/student/tests/test_email.py

```diff
@@ -94,6 +99,9 @@ class ActivationEmailTests(TestCase):
 
     def setUp(self):
         super(ActivationEmailTests, self).setUp()
+        self.inactive_user = UserFactory()
```

Comment:
Actually, there is a way you can create an inactive user as follows.
` UserFactory(is_active=False)`